package.json

```diff
@@ -5,19 +5,20 @@
     "build": "next build && npm run build:uiScripts",
     "build:accounts": "lerna run --scope @tidb-community/accounts build",
     "build:accounts:staging": "lerna run --scope @tidb-community/accounts build:staging",
-    "build:shared": "lerna run --ignore @tidb-community/accounts --ignore @tidb-community/ui-scripts build",
+    "build:shared": "lerna run --ignore @tidb-community/accounts --ignore @tidb-community/ui-scripts --ignore @tidb-community/strapi build",
+    "build:strapi": "lerna run --scope @tidb-community/strapi build",
     "build:uiScripts": "lerna run --scope @tidb-community/ui-scripts build",
     "build:watch": "lerna run --no-bail build:watch",
     "copy:locales": "rm -rf public/locales && cp -r packages/i18n/locales public",
     "deploy:production": "shipit production deploy",
     "deploy:preview": "shipit preview deploy",
-    "dev": "npm run build:shared && (npm run start:i18n & npm run start:mockApi & npm run strapi:dev & next dev)",
+    "dev": "npm run build:shared && npm run build:strapi && (npm run start:i18n & npm run start:mockApi & npm run strapi:dev & next dev)",
```

Comment:
`npm run strapi:dev` will build strapi in the development mode, so the `npm run build:strapi` command can be omitted.